app/views/games/show.html.erb

```diff
@@ -1,3 +1,70 @@
 <h1>Game Show page</h1>
 
-<p>Game name: <%= @game.name %></p>
\ No newline at end of file
+<p>Game name: <%= @game.name %></p> 
+
+<% array = ["A","B","C","D","E","F","G","H"] %> 
+<% unicodefig = ["&#9820;","&#9822;", "&#9821;", "&#9819;", "&#9818;", "&#9821;", "&#9822;", "&#9820;"] %>
+<table id = "ChessBoard"> 
+<% 8.times.each do |number| %> 
+	<% if number% 2 ==0 %>  
+		<tr> 
+			<% if number == 6 %> 
+				<% 8.times.each do |square| %>  
```

Comment:
Seems like we're duplicating a lot of effort here. Why not just create an `additional_class` variable or something, then assign the ``, `wp` or `bp` to it, then just interpolate with the static white/black class?

Also, what do the `wp` and `bp` mean? Kind of ambiguous naming IMO